examples/rebalancer/src/index.ts

```diff
@@ -29,6 +31,15 @@ const TRANSFER_GAS_LIMIT = 21000;
 const GAS_MULTIPLIER = 2;
 const ACTIVITIES_LIMIT = "100";
 
+// For demonstration purposes, create a globally accessible TurnkeyClient
+const turnkeyClient = new TurnkeyClient(
+  { baseUrl: process.env.BASE_URL! },
+  new ApiKeyStamper({
+    apiPublicKey: process.env.API_PUBLIC_KEY!,
+    apiPrivateKey: process.env.API_PRIVATE_KEY!,
+  })
+);
```

Comment:
I think this should be created inside of `main`? It's not very global if you have to pass it into each function anyway ;)

EDIT: oh I see. Actually it _is_ kind of global because it's not passed explicitly to `fundImpl` and `sweepImpl`

Reply:
Haha yeah maybe `global` wasn't the accurate term here but we can roll with it for now for the examples' sake!